dapps/marketplace/src/pages/listing/Listing.js

```diff
@@ -33,23 +34,30 @@ class Listing extends Component {
             </fbt>
           }
         />
-        <Query query={query} variables={vars}>
+        <Query query={query} variables={vars} errorPolicy="all">
           {({ networkStatus, error, data, refetch }) => {
+            const error404 = (
```

Comment:
Can we make this a component put at the top of the file? (ie outside the render method)